File: PAT (Basic-Level)/C/1053.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    int N, D;
    double e;
    scanf("%d %lf %d", &N, &e, &D);
    int proEmptyCount = 0, emptyCount = 0;
    for (int i = 0; i < N; ++i)
    {
        int obserDays;
        scanf("%d", &obserDays);
        int eleLessDays = 0;
        for (int j = 0; j < obserDays; ++j)
        {           
            double eleUsed;
            scanf("%lf", &eleUsed);
            if(eleUsed < e) eleLessDays++;
        }
        if(eleLessDays > (obserDays/2) && obserDays > D) emptyCount++;
        else if(eleLessDays > (obserDays/2) && obserDays <= D) proEmptyCount++;
    }
    printf("%.1f%% %.1f%%", (double)proEmptyCount/N * 100, (double)emptyCount/N * 100);
    return 0;
}

File: PAT (Basic-Level)/Python/1053.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

N, e, D = list(map(float, input().split()))
N = int(N)
D = int(D)
proEmptyCount = emptyCount = 0
for i in range(N):
    eleLessDays = 0
    eleUsed = list(map(float, input().split()))
    obserDays = int(eleUsed[0])
    for j in eleUsed[1:]:
        if j < e:
            eleLessDays += 1
    if eleLessDays > int(obserDays/2) and obserDays > D:
        emptyCount += 1
    elif eleLessDays > int(obserDays/2) and obserDays <= D:
        proEmptyCount += 1
print("%.1f%% %.1f%%" % (proEmptyCount/N*100, emptyCount/N*100))


'''edition2
N, e, D = list(map(float, input().split()))
N = int(N)
D = int(D) 
proEmptyCount = 0  
emptyCount = 0  
for i in range(N):  
    eleLessDays = 0  
    eleUsed = list(map(float, input().split()))
    obserDays = int(eleUsed[0])
    if obserDays <= D:  
        for j in eleUsed[1:]:  
            if j < e:  
                eleLessDays += 1  
        if eleLessDays*2 > obserDays:  
            proEmptyCount += 1  
    else:  
        for j in eleUsed[1:]:  
            if j < e:  
                eleLessDays += 1  
        if eleLessDays*2 > obserDays:  
            emptyCount += 1  
print("%.1f%% %.1f%%" % (proEmptyCount/N*100, emptyCount/N*100))  '''